src/admin-views/editor/metabox.php

```diff
@@ -21,15 +21,13 @@
 </div>
 
 <div id="event_tickets" class="eventtable" aria-live="polite">
-	<?php wp_nonce_field( 'tribe-tickets-meta-box', 'tribe-tickets-post-settings' ); ?>
-
-	<?php $admin_views->template( [ 'editor', 'panel', 'list' ], get_defined_vars() ); ?>
-
-	<?php $admin_views->template( [ 'editor', 'panel', 'ticket' ], get_defined_vars() ); ?>
+	<?php
+	wp_nonce_field( 'tribe-tickets-meta-box', 'tribe-tickets-post-settings' );
 
-	<?php $admin_views->template( [ 'editor', 'panel', 'settings' ], get_defined_vars() ); ?>
+	foreach ( $panels as $panel ) {
```

Comment:
A couple things:

1. How does this relate to the schema updates? I don't see it mentioned in the PR description.
2. `$panels` and `$ticket_id` aren't defined, or at least not that we know about at this point. They either need to be defined, or else declared at the top of the file with the other vars. (`$ticket_id` wasn't changed in this PR< but if we're adding to the file vars anyway then we should include it now)